sky/packages/sky/lib/src/animation/performance.dart

```diff
@@ -51,30 +51,30 @@ abstract class WatchableAnimationPerformance {
 /// may also take direct control of the timeline by manipulating [progress], or
 /// [fling] the timeline causing a physics-based simulation to take over the
 /// progression.
-class AnimationPerformance implements WatchableAnimationPerformance {
-  AnimationPerformance({ this.duration, double progress }) {
+class Performance implements PerformanceView {
```

Comment:
Specifically, the problem with Animation is that it is very easy to confuse with all the other terms in this space: Animatable, AnimatedValue, Animation... and what would you call ValuePerformance?